### PR TITLE
Added details on how to configure Entra OIDC

### PR DESCRIPTION
**Pull request by mkaesz-wandb**
## Description
Added details on how to configure Entra OIDC

## Ticket
No

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [X] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [X] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [X] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/hosting/iam/sso.md [hunk 1]
```diff
@@ -133,6 +130,66 @@ the redirect URI would look like: `https://localhost:8080/oidc/callback`.
...
 
   </TabItem>
+
+<TabItem value="entra">
+1. Login to the [Azure Portal](https://portal.azure.com/#home).
+
+2. Select "Microsoft Entra ID" service.
+![](/images/hosting/entra_select_entra_service.png)
+
+3. On the left side, select "App registrations".
```
Comment (johndmulhausen) [inline docs/guides/hosting/iam/sso.md]:
```suggestion
3. On the left side, select "App registrations."
```

File: docs/guides/hosting/iam/sso.md [hunk 2]
```diff
@@ -17,15 +17,12 @@ W&B Server operates with manual user management by default. Licensed versions of
...
-:::tip
-Select either a Single Page or Public Client application in your identity provider (IdP).
-:::
-
 <Tabs
   defaultValue="aws"
   values={[
     {label: 'AWS', value: 'aws'},
     {label: 'Okta', value: 'okta'},
+     {label: 'Entra', value: 'entra'},
```
Comment (johndmulhausen) [inline docs/guides/hosting/iam/sso.md]:
```suggestion
    {label: 'Okta', value: 'okta'},
    {label: 'Entra', value: 'entra'},
```

File: docs/guides/hosting/iam/sso.md [hunk 3]
```diff
@@ -133,6 +130,66 @@ the redirect URI would look like: `https://localhost:8080/oidc/callback`.
...
+<TabItem value="entra">
+1. Login to the [Azure Portal](https://portal.azure.com/#home).
+
+2. Select "Microsoft Entra ID" service.
+![](/images/hosting/entra_select_entra_service.png)
+
+3. On the left side, select "App registrations".
+![](/images/hosting/entra_app_registrations.png)
+
+4. On the top, click "New registration".
```
Comment (johndmulhausen) [inline docs/guides/hosting/iam/sso.md]:
```suggestion
4. On the top, click "New registration."
```

File: docs/guides/hosting/iam/sso.md [hunk 4]
```diff
@@ -133,6 +130,66 @@ the redirect URI would look like: `https://localhost:8080/oidc/callback`.
...
+![](/images/hosting/entra_new_app_registration.png)
+
+    On the screen named "Register an application", fill out the values as follows:
```
Comment (johndmulhausen) [inline docs/guides/hosting/iam/sso.md]:
```suggestion
    On the screen named "Register an application," fill out the values as follows:
```

File: docs/guides/hosting/iam/sso.md [hunk 5]
```diff
@@ -133,6 +130,66 @@ the redirect URI would look like: `https://localhost:8080/oidc/callback`.
...
+![](/images/hosting/entra_register_an_application.png)
+
+    - Specify a name, e.g. "Weights and Biases application".
```
Comment (johndmulhausen) [inline docs/guides/hosting/iam/sso.md]:
```suggestion
    - Specify a name, for example "Weights and Biases application".
```

File: docs/guides/hosting/iam/sso.md [hunk 6]
```diff
@@ -133,6 +130,66 @@ the redirect URI would look like: `https://localhost:8080/oidc/callback`.
...
+    - Leave the default account type: "Accounts in this organizational directory only (Default Directory only - Single tenant)". Modify if you need to.
```
Comment (johndmulhausen) [inline docs/guides/hosting/iam/sso.md]:
```suggestion
    - Leave the default account type: "Accounts in this organizational directory only (Default Directory only - Single tenant)." Modify if you need to.
```

Comment (johndmulhausen) [reply docs/guides/hosting/iam/sso.md]: @mkaesz-wandb "Leave the default" and "modify if you need to" are contradictory. Which is it?

File: docs/guides/hosting/iam/sso.md [hunk 7]
```diff
@@ -133,6 +130,66 @@ the redirect URI would look like: `https://localhost:8080/oidc/callback`.
...
+    - Configure Redirect URI as type "Web" with value: `https://YOUR_W_AND_B_URL/oidc/callback`
+    - Click "Register".
```
Comment (johndmulhausen) [inline docs/guides/hosting/iam/sso.md]:
```suggestion
    - Click "Register."
```

File: docs/guides/hosting/iam/sso.md [hunk 8]
```diff
@@ -133,6 +130,66 @@ the redirect URI would look like: `https://localhost:8080/oidc/callback`.
...
+
+- Make a note of the "Application (client) ID" and "Directory (tenant) ID".
```
Comment (johndmulhausen) [inline docs/guides/hosting/iam/sso.md]:
```suggestion
- Make a note of the "Application (client) ID" and "Directory (tenant) ID."
```

File: docs/guides/hosting/iam/sso.md [hunk 9]
```diff
@@ -133,6 +130,66 @@ the redirect URI would look like: `https://localhost:8080/oidc/callback`.
...
+    - Click "Register".
+
+- Make a note of the "Application (client) ID" and "Directory (tenant) ID". 
+![](/images/hosting/entra_app_overview_make_note.png)
+
+
+5. On the left side, click "Authentication"
+![](/images/hosting/entra_select_authentication.png)
+
+- Under "Front-channel logout URL" specify: `https://YOUR_W_AND_B_URL/logout`
```
Comment (johndmulhausen) [inline docs/guides/hosting/iam/sso.md]:
```suggestion
- Under **Front-channel logout URL**, specify: `https://YOUR_W_AND_B_URL/logout`
```

File: docs/guides/hosting/iam/sso.md [hunk 10]
```diff
@@ -133,6 +130,66 @@ the redirect URI would look like: `https://localhost:8080/oidc/callback`.
...
+    - Specify a name, e.g. "Weights and Biases application".
+    - Leave the default account type: "Accounts in this organizational directory only (Default Directory only - Single tenant)". Modify if you need to.
+    - Configure Redirect URI as type "Web" with value: `https://YOUR_W_AND_B_URL/oidc/callback`
+    - Click "Register".
+
+- Make a note of the "Application (client) ID" and "Directory (tenant) ID". 
+![](/images/hosting/entra_app_overview_make_note.png)
+
+
+5. On the left side, click "Authentication"
```
Comment (johndmulhausen) [inline docs/guides/hosting/iam/sso.md]:
```suggestion
5. On the left side, click **Authentication**.
```

**Review by johndmulhausen — CHANGES_REQUESTED** [review]

**Comment by johndmulhausen** [comment]
Please finish responding to the Vale comments as I suggested. Also, note that when quoting the text of UI elements you should use `**bold**` text and not quotation marks.

**Review by ngrayluna — APPROVED** [review]

**Pull request merged**